core/reactor.c

```diff
@@ -194,7 +200,7 @@ int _lf_do_step(void) {
 #endif
 
     if (lf_tag_compare(current_tag, stop_tag) >= 0) {
-        return 0;
+        exit(0);
```

Comment:
What is the reason for this change? It looks like the code should do the same thing either way, but the doc comments do not say anything about returning with 0 exit code directly out of this function.